py4cast/models/vision/unetrpp.py

```diff
@@ -125,17 +126,18 @@ def __init__(
         self,
         input_size: int,
         hidden_size: int,
-        proj_size: int,
         num_heads: int,
         dropout_rate: float = 0.0,
         pos_embed=False,
         spatial_dims=2,
+        proj_size: int = 64,
+        use_scaled_dot_product_CA: bool = True,
+        use_scaled_dot_product_SA: bool = True,
     ) -> None:
         """
         Args:
             input_size: the size of the input for each stage.
             hidden_size: dimension of hidden layer.
-            proj_size: projection size for keys and values in the spatial attention module.
             num_heads: number of attention heads.
             dropout_rate: faction of the input units to drop.
             pos_embed: bool argument to determine if positional embedding is used.
```

Comment:
Can you please update doc with the added bools ?

Reply:
Done